openshift/catalog/openshift/openshift.bom

```diff
@@ -0,0 +1,720 @@
+                tee ~/.ssh/config <<-EOF
+                IdentityFile=~/.ssh/openshift.pem
+                EOF
+                chmod 600 ~/.ssh/openshift.pem ~/.ssh/config
+                ssh-keygen -y -f ~/.ssh/openshift.pem >> ~/.ssh/authorized_keys
+              launch.command: |
+                cd ${RUN_DIR}/openshift-ansible
+                ansible-playbook playbooks/byo/config.yml --inventory ${INSTALL_DIR}/inventory
+              checkRunning.command: |
+                which ansible-playbook
+              stop.command: |
+                cd ${RUN_DIR}/openshift-ansible
+                ansible-playbook playbooks/adhoc/uninstall.yml --inventory ${INSTALL_DIR}/inventory
+
+            brooklyn.enrichers:
+              - type: org.apache.brooklyn.enricher.stock.Propagator
+                brooklyn.config:
+                  producer: $brooklyn:entity("openshift-master-machines")
+                  sensorMapping:
+                    $brooklyn:sensor("hosts.list"): $brooklyn:sensor("master.hosts.list")
+              - type: org.apache.brooklyn.enricher.stock.Propagator
+                brooklyn.config:
+                  producer: $brooklyn:entity("openshift-etcd-machines")
+                  sensorMapping:
+                    $brooklyn:sensor("hosts.list"): $brooklyn:sensor("etcd.hosts.list")
+              - type: org.apache.brooklyn.enricher.stock.Propagator
+                brooklyn.config:
+                  producer: $brooklyn:entity("openshift-node-machines")
+                  sensorMapping:
+                    $brooklyn:sensor("hosts.list"): $brooklyn:sensor("node.hosts.list")
+              - type: org.apache.brooklyn.enricher.stock.Propagator
+                brooklyn.config:
+                  producer: $brooklyn:entity("openshift-lb")
+                  propagating:
+                    - $brooklyn:sensor("lb.entity.id")
+              - type: org.apache.brooklyn.enricher.stock.Propagator
+                brooklyn.config:
+                  producer: $brooklyn:root()
+                  sensorMapping:
+                    $brooklyn:sensor("service.isUp"): $brooklyn:sensor("scheduler.start")
+              - type: org.apache.brooklyn.enricher.stock.Propagator
+                brooklyn.config:
+                  producer: $brooklyn:entity("openshift-lb")
+                  propagating:
+                    - $brooklyn:sensor("host.address")
+              - type: org.apache.brooklyn.enricher.stock.Transformer
+                brooklyn.config:
+                  enricher.triggerSensors:
+                    - host.address
+                  enricher.targetSensor: $brooklyn:sensor("main.uri")
+                  enricher.targetValue:
+                    $brooklyn:formatString:
+                      - "https://%s:8443/"
+                      - $brooklyn:attributeWhenReady("host.address")
+              - type: io.cloudsoft.amp.ui.common.enricher.AmpExternalUiModuleEnricher
```

Comment:
are we using this? if not can it be removed - deploying with vanilla Brooklyn requires it to be removed